src/AstroBase.jl

```diff
@@ -485,4 +487,85 @@ function xy06(jd1, jd2)
     sec2rad((xpr + (xyls[1] + xypl[1]) / 1e6)), sec2rad(ypr + (xyls[2] + xypl[2]) / 1e6)
 end
 
+"""
+    nutation_00a(jd1, jd2)
+
+Returns luni-solar and planetary nutations for a given 2 part Julian date(TT).
+
+# Example
+
+```jldoctest
+julia> nutation_00a(2.4578265e6, 0.30440190993249416)
+(-3.7589903391357206e-5, -3.6659049617818334e-5)
+```
+"""
+function nutation_00a(jd1, jd2)
+    U2R = sec2rad(1/1e7)
+    t = ((jd1 - J2000) + jd2) / DAYS_PER_CENTURY
+
+    el = mean_anomaly(luna, t)
+    f  = mean_longitude_minus_lan(luna, t)
+    om = mean_longitude_ascending_node(luna, t)
+    apa  = general_precession_in_longitude(t)
+    alme = mean_longitude(mercury, t)
+    alve = mean_longitude(venus, t)
+    alea = mean_longitude(earth, t)
+    alma = mean_longitude(mars, t)
+    alju = mean_longitude(jupiter, t)
+    alsa = mean_longitude(saturn, t)
+    alur = mean_longitude(uranus, t)
+
+    elp = sec2rad(mod((@evalpoly t 1287104.79305 129596581.0481 -0.5532 0.000136 -0.00001149), ARCSECONDS_IN_CIRCLE))
+    d   = sec2rad(mod((@evalpoly t 1072260.70369 1602961601.2090 -6.3706 0.006593 -0.00003169), ARCSECONDS_IN_CIRCLE))
+
+    al   = mod2pi(2.35555598 + 8328.6914269554 * t)
+    af   = mod2pi(1.627905234 + 8433.466158131 * t)
+    ad   = mod2pi(5.198466741 + 7771.3771468121 * t)
+    aom  = mod2pi(2.18243920 - 33.757045 * t)
+    alne = mod2pi(5.321159000 + 3.8127774000 * t)
+
+    dp = 0.0
+    de = 0.0
+
+    for i in reverse(eachindex(xls_nutation))
+        arg = mod(xls_nutation[i][1]  * el +
+                    xls_nutation[i][2] * elp +
+                    xls_nutation[i][3]  * f +
+                    xls_nutation[i][4]  * d +
+                    xls_nutation[i][5] * om, 2pi)
+        sarg, carg = sincos(arg)
+        dp += (xls_nutation[i][6] + xls_nutation[i][7] * t) * sarg + xls_nutation[i][8] * carg
+        de += (xls_nutation[i][9] + xls_nutation[i][10] * t) * carg + xls_nutation[i][11] * sarg
+    end
+
+    dpsils, depsls = dp * U2R, de * U2R
```

Comment:
I would prefer this expression: `dpsils, depsls = sec2rad(dp * 1e-7), sec2rad(de * 1e-7)`

This is more characters in total but the intention is clear from looking at the line. If you introduce the constant I need to scroll up to understand what it does.